src/services/loggingexport/loggingexportService.js

```diff
@@ -0,0 +1,172 @@
+ * 
+ * Look for //TODO: comments for where to customize the service to your needs
+ * 
+ * N.B. Be wary of adding too many Logger messages within this service as it will of course receive
+ * back to transmit onwards
+ */
+const Finsemble = require("@chartiq/finsemble");
+const RouterClient = Finsemble.Clients.RouterClient;
+const Logger = Finsemble.Clients.Logger;
+const Request = require("superagent");
+Logger.start();
+
+// Add and initialize any other clients you need to use 
+//   (services are initialised by the system, clients are not)
+// let ConfigClient = Finsemble.Clients.ConfigClient;
+// ConfigClient.initialize();
+
+// Settings 
+/** Transmit queued messages when their number reaches or exceeds this amount. */
+const BATCH_SIZE = 100;
+/** Wait at most this number of milliseconds after first message in batch is received before transmitting. */
+const TIMEOUT_MILLISECS = 30 * 1000;
+/** Log levels of messsages to capture. Valid values: Error, Warning, Log, Info, Debug, Verbose */
+const CAPTURE_LOG_LEVELS = {
+	"Log": true, 
+	"Warn": true, 
+	"Error": true
+};
+/** Log categories of messages to capture. Valid values: system, dev or perf */
+const CAPTURE_LOG_CATEGORIES = { 
+	"system": true, 
+	"dev": true
+};
+/** Where to transmit the logs to */
+//TODO: Update to your logging endpoint
+const LOGGING_ENDPOINT = "http://somedomain.com/loggingendpoint";
+
+
+/**
+ * Log messages arrive in small batches from each logger and must be added to an export batch and 
+ * transmitted. 
+ * 
+ * @constructor
+ */
+function loggingexportService() {
+	const self = this;
+	let timeout = null;
+	let batchAllocSize = BATCH_SIZE + 10;     //add a small amount of leeway on batch size
+	let logBatch = new Array(batchAllocSize); //pre-init the batch array to reduce array allocations
+	let currBatchSize = 0;
+	let msgTmp
+
+	/**
+	 * Add an incoming array of log messages to the batch and transmit if thresholds met.
+	 * 
+	 * Logging message format:
+	 *		[										
+	 *			{
+	 *				"category": "system",				//Log message type: system, dev or perf
+	 *				"logClientName": "Finsemble",		//The registered name of the logger instance
+	 *				"logType": "Log",					//Log level: Error, Warning, Log, Info, Debug, Verbose
+	 *				"logData": "[\"SERVICE LIFECYCLE: STATE CHANGE: Service initializing\",\"windowService\"]",
+	 *													//JSON encoded erray of message and data components of the log message
+	 *													//N.B. maybe be prefixed by string "*** Logging Error: ""
+	 *				"logTimestamp": 1544090028391.6226	//Log message timestamp for ordering use
+	 *			}, 
+	 *			{...},
+	 *			...
+	 *		]
+	 * @private
+	 */
+	this.addToBatch = function (dataArr) {
+		if (dataArr) {
+			for (let m = 0; m < dataArr.length; m++) {
+				if (CAPTURE_LOG_CATEGORIES[dataArr[m].category] && CAPTURE_LOG_LEVELS[dataArr[m].logType]) {
+					//TODO: add any necessary message format changes here
+					logBatch[currBatchSize++] = dataArr[m];
+				} 
+				// else {
```

Comment:
Would it be useful to keep this message as a `console.debug`? If not, remove the commented out code.